### ISSUE TITLE
PropBlocks - ready to use Open Prop blocks

### ISSUE THREAD

**Issue by argyleink** [argyleink/open-props#153]
one of many Tailwind examples to be inspired by https://github.com/mertJF/tailblocks

**Comment by eddsaura** [comment]
Hey, I'd love to contribute. Do you want to make a list of interesting blocks whatever?

**Comment by argyleink** [comment]
pretty much [all of these here](https://tailblocks.cc/) could be made with Open Props. 
could start for now in codepen or something, ensure the blocks are responsive and dark/light adaptive, and eventually make a site to share them? these codepens could get promoted / added to the "Built with Open Props" section of the OP site while a bigger aggregated site is made?

**Comment by hunzaboy** [comment]
The codepen sounds better. I will be contributing to this too :) . FYI the media queries ( custom media ) are not working in codepen , even using the postcss-custom-media. Any idea about it @argyleink ?

**Comment by argyleink** [comment]
> FYI the media queries ( custom media ) are not working in codepen , even using the postcss-custom-media. Any idea about it @argyleink ? 

Yeah, I talked to Codepen about it a month ago or so, currently not possible (at least in any reasonable ways). Forgot about that! Stackblitz could be used instead, which also means all the blocks could be using jit-props too, which would be great for helping that workflow be seen more. Let's shift to stackblitz then!? Which also means we should be thoughtful about #112.

Thanks for looking into this and offering help!

**Comment by codevalve** [comment]
This sounds great!  Has there been any movement on this?

**Comment by argyleink** [comment]
not blocks specifically yet (tho there's some codepens on the website homepage to use).
more effort has been going into [buttons](https://codepen.io/argyleink/pen/gOXbvjj?editors=1100) and recently forms https://codepen.io/argyleink/pen/RwxJKQa/2256560bf93a131b58b5774e840d51f0
